CEUS/CEUS_download.py

```diff
@@ -0,0 +1,39 @@
+"""
+This module downloads Commercial End-Use Survey (CEUS) files into intended filepath.
+"""
+
+import requests
+
+def CEUS_download(FCZs, buildingtypes, ceus_url, filepath):
```

Comment:
Is there anything unique about CEUS download v.s. RBSA downloads - or any future downloads for that matter?

We can make this generic where a list of URLs get passed in and the logic that is specific to CEUS remains the onus of the caller of the download function. A download file class doesn't need to know about CEUS filename construction.